modules/fio-uninstall-console.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * security/file_integrity_operator/fio-uninstalling.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="fio-uninstall-console_{context}"]
+= Uninstall the File Integrity Operator using the web console
+
+To remove the File Integrity Operator, you must first delete the `FileIntegrity` objects in the FIO namespace. After the objects are removed, you can then remove the Operator and its namespace.
```

Comment:
According to the [Official Red Hat product names list](https://docs.google.com/spreadsheets/d/1DLS_lS3VKidgZIvcLmLp9BoiqptkvqHWfe1D5FD2kfk/edit?gid=1375785039#gid=1375785039&range=F113), FIO is not approved for use in place of File Integrity Operator. 
```suggestion
To remove the File Integrity Operator, you must first delete the `FileIntegrity` objects in the namespace. After the objects are removed, you can then remove the Operator and its namespace.
```